README.md

```diff
@@ -1,138 +1,247 @@
+Trino client
```

Comment:
nit: Trino Python client

Reply:
DONE. I think we need to change title of [trino-go-client](https://github.com/trinodb/trino-go-client) as well